README.md

```diff
@@ -95,7 +95,8 @@ Plugin Options
 Browser extensions
 ------------------
 This input tool is also available as a Chrome extension:
```

Comment:
Can you update this line to: 

This input tool is also available as a browser extension:

Reply:
@kartikm done